index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('Ingrese su nombre')
+alert(`Hola ${Usuario}`);
```

Comment:
En este caso  me parecería mejor que no uses interpolación de strings
```suggestion
alert('Hola' + nombreUsuario);
```